mkDocs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+## CREATE TARGET DIRECTORY ##
+rm -rf target/generated-docs
```

Comment:
replace 

```shell
rm -rf target/generated-docs
mkdir target/generated-docs
```
by:
```shell
mkdir -p target/generated-docs # if dir exists, nothing created, otherwise, it creates the dir, and parent if needed. 
rm -rf target/generated-docs/*  # remove content of the dir, but do not delete the dir itself
```